Add Support for Linux Mint

trevormerritt opened the pull request:
Add support for Linux Mint
Linux Mint 20.x = Ubuntu Focal
Linux Mint 21.x = Ubuntu Jammy
Linux Mint DE 6 = Debian 12

<!--

Thank you for contributing to this project!

If you need any help please feel free to contact us on Discord: https://discord.com/invite/uCPdDXzbdv
Or, mention our core members by typing `@GitHub_Handle` on any issue / PR

Add some test cases! It help reviewers to understand the behaviour and prevent it to be broken in the future.

-->

## PR Info

<!-- mention the related issue -->
- Closes  #42
<!-- is this PR depends on other PR? (if applicable) -->
- Dependencies:
None

<!-- any PR depends on this PR? (if applicable) -->
- Dependents:
None

## New Features

- [ Add support for Linux Mint 20.x ]
- [ Add support for Linux Mint 21.x ]
- [ Add support for Linux Mint Debian Edition 6 ]

## Bug Fixes

- [ N/A ] <!-- if it fixes a bug, please provide a brief analysis of the original bug -->

## Breaking Changes

- [ N/A ] <!-- any change in behaviour or method signature? is it backward compatible? -->

## Changes

- [ Adds os_like variable to install script which is used to determine which Linux Mint variant is in use.  Value is located in the same file as the other values used for determining the host platform]

billy1624 commented:
- Linux Mint 21.3 <img width="1136" alt="image" src="https://github.com/SeaQL/FireDBG.for.Rust/assets/30400950/cf82d6e3-50a3-4cdd-ba6f-271d0dabc45b">

tyt2y3 reviewed (approved):
Thank you

billy1624 commented:
Hey @trevormerritt, could you please try installing FireDBG on your Linux Mint?

```sh
curl https://raw.githubusercontent.com/SeaQL/FireDBG.for.Rust/main/install.sh -sSf | sh
```

trevormerritt commented:
I tested 2 systems today.  One system was a non-dev workstation on Mint 21.3 that I added 'build-essentials' and did a run of the 'rustup' install.  The second system was my dev workstation I use at work on LMDE 6.  I moved the ~/.cargo directory and re-ran the 'rustup' install to have a clean instance to start with.

Once the prep steps outlined above were completed, both systems installed and ran the tests successfully.

```
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
curl https://raw.githubusercontent.com/SeaQL/FireDBG.for.Rust/main/install.sh -sSf | sh
```


![image](https://github.com/SeaQL/FireDBG.for.Rust/assets/3418965/e5ef6d83-1c88-4847-a183-f4d47de3b0e1)
![image](https://github.com/SeaQL/FireDBG.for.Rust/assets/3418965/df68bac2-1104-4446-a410-7078464b10c0)

billy1624 commented:
Perfect! Thanks for the updates! @trevormerritt Hope you enjoy using FireDBG :P
